app/public.js

```diff
@@ -281,6 +282,20 @@
     }
   }
 
+  function getName(s) {
+    let n = s.replace(/^.*[\\\/]/, '');
+    if (n.indexOf('?')>0) {
```

Comment:
This is just to render the filename in the overlay, removing query strings makes the filename more readable and the href is still available for the full image URL.